frontend/pages/hosts/details/cards/Policies/HostPoliciesTable/PolicyFailingCount/PolicyFailingCount.tsx

```diff
@@ -5,11 +5,14 @@ import IssueIcon from "../../../../../../../../assets/images/icon-issue-fleet-bl
 
 const baseClass = "policy-failing-count";
 
-const PolicyFailingCount = (policyProps: {
+interface IPolicyFailingCountProps {
```

Comment:
Fix to match FE patterns of using interfaces for prop types

Reply:
nice 👍🏽